mui/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 3.0.0-beta.0 (Not yet published)
```

Comment:
We don't list out the alphas and betas in the Changelog. We lump these under what the version is going to be and the "Not Published Yet" message indicates that it is still in pre-release status.

```suggestion
## 3.0.0 (Not yet published)
```